fsw/src/sample_app.c

```diff
@@ -26,65 +26,86 @@
 *******************************************************************************/
 
 /*
-**   Include Files:
+** Include Files:
 */
-
 #include "sample_app.h"
-#include "sample_app_perfids.h"
-#include "sample_app_msgids.h"
-#include "sample_app_msg.h"
-#include "sample_app_events.h"
-#include "sample_app_version.h"
```

Comment:
I'd expect these to be here instead of sample_app.h.  Is this what the app developers guide recommends?